byterun/caml/callback.h

```diff
@@ -43,9 +41,13 @@ CAMLextern value caml_callbackN_exn (value closure, int narg, value args[]);
 #define Is_exception_result(v) (((v) & 3) == 2)
 #define Extract_exception(v) ((v) & ~3)
 
+#if CAML_API_VERSION < 405
```

Comment:
Since we missed the 4.05 deadline, this should be changed to `406`.
You'll have to grep `405` to find all occurrences.